lectures/inflation_history.md

```diff
@@ -372,7 +373,7 @@ for i in range(4):
     sheet_list = [process_df(df) for df in sheet_list]
     df_list.append(pd.concat(sheet_list, axis=1))
 
-df_Aus, df_Hung, df_Pol, df_Germ = df_list
+df_aus, df_hung, df_pol, df_germ = df_list
```

Comment:
@Jiarui-ZH I tend to use `iso3c` codes when using shorter versions for countries in code. So I would use `df_aus, df_hun, df_pol, df_deu`

https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3